task/drivers/cgi/driver.go

```diff
@@ -31,12 +32,13 @@ type Config struct {
 }
 
 // New returns the task execution driver.
-func New(d downloader.Downloader) task.Handler {
-	return &driver{downloader: d}
+func New(d downloader.Downloader, pl packaged.PackageLoader) task.Handler {
```

Comment:
Just one question: is this used only by runner?

Reply:
yes